be/src/formats/parquet/column_chunk_reader.cpp

```diff
@@ -51,19 +52,56 @@ Status ColumnChunkReader::init(int chunk_size) {
     RETURN_IF_ERROR(get_block_compression_codec(compress_type, &_compress_codec));
 
     RETURN_IF_ERROR(_try_load_dictionary(chunk_size));
-    RETURN_IF_ERROR(_parse_page_data());
     return Status::OK();
 }
 
 Status ColumnChunkReader::next_page() {
```

Comment:
`load_header` is more accurate

Reply:
Yes. Or rename `_parse_page_header` to `load_page_header` to save a function.